src/modules/funtions.js

```diff
@@ -0,0 +1,20 @@
+const Tasks = JSON.parse(localStorage.getItem('list')) || [];
+const input = document.querySelector('.inp');
+const Add = () => {
+  const create = {
+    description: input.value,
+    completed: false,
+    index: Tasks.length,
```

Comment:
- [ ] Good job so far. Kindly note that the project requirement expects that the index of the todos should start from one (1) such that when a fifth todo is added, its index gives 5. please  update your index to start from 1

![image](https://user-images.githubusercontent.com/63473584/228654749-587ca110-e732-4f21-9172-31863674341e.png)